proposals/0296-Update-video-streaming-capabilities-during-ignition-cycle.md

```diff
@@ -118,10 +118,32 @@ While `diagonalScreenSize` is mandatory, `aspectRatio` and `resolution` are opti
 In `SDLStreamingMediaConfiguration`:
 
 ```objectivec
-@property (strong, nonatomic, nullable) VideoStreamingRange *supportedLandscapeStreamingRange;
-@property (strong, nonatomic, nullable) VideoStreamingRange *supportedPortraitStreamingRange;
+@property (strong, nonatomic, nullable) SDLVideoStreamingRange *supportedLandscapeStreamingRange;
+@property (strong, nonatomic, nullable) SDLVideoStreamingRange *supportedPortraitStreamingRange;
+@property (weak, nonatomic, nullable) id<SDLStreamingMediaDelegate> delegate;
+```
+
+Where an object conforming to `SDLStreamingMediaDelegate` should implement the following methods:
+
+```objectivec
+@protocol SDLStreamingMediaDelegate <NSObject>
 
-@interface VideoStreamingRange : NSObject
+- (void)videoStreamingSizeDidUpdate:(CGSize)displaySize;
+- (void)videoStreamingSizeDoesNotMatch;
+
+@end
+```
+
+Whenever the video streaming capability changes `SDLStreamingVideoLifecycleManager` should call on the delegate:
+```objectivec
+[weakSelf.delegate videoStreamingSizeDidUpdate:displaySize];
+```
+In this case the client application will have a chance to update its view accordingly once it implements the delegate method.
+
+`SDLVideoStreamingRange` class should have the following properties:
+
+```objectivec
+@interface SDLVideoStreamingRange : NSObject
```

Comment:
I do not see this included in https://github.com/smartdevicelink/sdl_evolution/issues/983#issuecomment-642797352.  Please remove, and request as a new revision, or advise if it's in the comment and I missed it.  Thanks!

Reply:
@theresalech This is just renaming of existing `SDLSupportedStreamingRange` to `SDLVideoStreamingRange` as suggested in https://github.com/smartdevicelink/sdl_evolution/issues/983#issuecomment-658826750 for item 3

Reply:
Hello @theresalech, let me explain.
There was a question:
[3.2. How do we notify the app dev on changes?](https://github.com/smartdevicelink/sdl_evolution/issues/983#issuecomment-641424826)
and the [answer](https://github.com/smartdevicelink/sdl_evolution/issues/983#issuecomment-642797352)
The code above (lines 137-141) implements the requested functionality.

Reply:
@yoooriii @dboltovskyi it doesn't appear this was part of what was agreed to by the Steering Committee: https://github.com/smartdevicelink/sdl_evolution/issues/983#issuecomment-658826750.  Please remove and include in your request for additional revisions.  Thank you!

Reply:
@theresalech 
Current update includes the following (as suggested in https://github.com/smartdevicelink/sdl_evolution/issues/983#issuecomment-658826750 for item 3):

1.
> For iOS, change streamingVideoDelegate to delegate.

2. 
> For Both iOS and Java Suite, use VideoStreamingRange

Please advise what part of the changes should be removed.

Reply:
The comment detailing Steering Committee vote includes the new class names, but not the logic as described in lines 126 - 140.  You are correct that `VideoStreamingRange` is included in the comment, so lines 143 - 146 can remain as is.  Thanks!

Reply:
@theresalech Thank you for the clarification. Please find update in 32adadc